addons/l10n_nz/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="tax_report_sale_and_income" model="account.tax.report.line">
+        <field name="name">Sales and Income</field>
+        <field name="sequence" eval="1"/>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box5" model="account.tax.report.line">
+        <field name="name">[BOX 5] Total sales and income for the period(including GST and zero-rated Supplies)</field>
+        <field name="tag_name">BOX 5</field>
+        <field name="parent_id" ref="tax_report_sale_and_income"/>
+        <field name="sequence" eval="1"/>
+        <field name="code">BOX5</field>
+        <field name="country_id" ref="base.nz"/>
+    </record>
+
+    <record id="tax_report_box6" model="account.tax.report.line">
+        <field name="name">[BOX 6] Zero-rated supplies in Box 5</field>
+        <field name="tag_name">BOX 6</field>
+        <field name="parent_id" ref="tax_report_sale_and_income"/>
+        <field name="sequence" eval="2"/>
+        <field name="code">BOX6</field>
```

Comment:
Is it possible to make this name more unique?  (e.g. NZBOX6, check other localizations...)